static/bidder-info/yandex.yaml

```diff
@@ -0,0 +1,13 @@
+endpoint: "https://bs-metadsp.yandex.ru/prebid/{{.PublisherID}}?ssp-id=10500"
```

Comment:
@chernodub  PBS recommends to use static domain. Is it possible to use query param.

```
https://bs-metadsp.yandex.ru/prebid?PublisherID={{.PublisherID}}&&ssp-id=10500"

```

Reply:
Looks reasonable, but unfortunately this will make us rewrite the logic in both our PBS Java and Prebid.js adapters to make the behavior consistent. Maybe we could later open PRs in all the adapters at the same time. Would that be possible?

Also, as far as I see in the module rules, it is only required to make the domain static, not the path:
> [Endpoint domain names cannot be fully variable. At some point, they won’t be able contain any variables at all. e.g. $PARAM1.example.com.](https://docs.prebid.org/dev-docs/module-rules.html#:~:text=Endpoint%20domain%20names%20cannot%20be%20fully%20variable.%20At%20some%20point%2C%20they%20won%E2%80%99t%20be%20able%20contain%20any%20variables%20at%20all.%20e.g.%20%24PARAM1.example.com.)

Reply:
By the way, my team helped me figure that `PublisherID` doesn't really represent the entity we meant to use here, so I will update the macros to something else.

Moreover, @onkarvhanumante, is it possible to extend [`EndpointTemplateParams`](https://github.com/prebid/prebid-server/blob/b806a53b4b8045b106004ac7d86700368c619582/macros/macros.go) with bidder-specific values? We need something like `PageID` (in our domain logic, it repesents one of the publisher's domains/apps)

Reply:
> By the way, my team helped me figure that `PublisherID` doesn't really represent the entity we meant to use here, so I will update the macros to something else.
> 
> Moreover, @onkarvhanumante, is it possible to extend [`EndpointTemplateParams`](https://github.com/prebid/prebid-server/blob/b806a53b4b8045b106004ac7d86700368c619582/macros/macros.go) with bidder-specific values? We need something like `PageID` (in our domain logic, it repesents one of the publisher's domains/apps). If not, I'm thinking of using `zoneId` or `sourceId` as an alias, as it seems related.

you can add new field `PageID` in [EndpointTemplateParams](https://github.com/prebid/prebid-server/blob/b806a53b4b8045b106004ac7d86700368c619582/macros/macros.go)